packages/fxa-components/Portal/index.scss

```diff
@@ -0,0 +1,24 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+.portal#top-bar { // scss-lint:disable IdSelector
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  z-index: 9999;
+}
+
+.portal#dialogs { // scss-lint:disable IdSelector
```

Comment:
Don't need a class if we're using an ID

Reply:
Update - since this getting ripped out soon don't let this block you

Reply:
Yeah, I'm going to address this in the follow up, it's leftover from payments so I just left it as-is.